### PR TITLE
Change placeholder name to be more clear

### PR DESCRIPTION
**Pull request by rakekniven**
Reported at Transifex.

Signed-off-by: rakekniven <2069590+rakekniven@users.noreply.github.com>

### CODE REVIEW

**Comment by rakekniven** [comment]
Fix for #2702

**Review by raimund-schluessler — APPROVED** [review]

**Comment by rakekniven** [comment]
@eneiluj Can you please review.

File: src/components/DashboardWidget/DashboardWidget.vue [hunk 1]
```diff
@@ -316,7 +316,7 @@ export default {
 		},
 
 		showMoreLabel() {
-			return t('More {what} …', { what: this.showMoreText })
+			return t('More {dashboardItemType} …', { what: this.showMoreText })
```
Comment (korelstar) [inline src/components/DashboardWidget/DashboardWidget.vue:319]:
```suggestion
			return t('More {dashboardItemType} …', { dashboardItemType: this.showMoreText })
```

**Review by korelstar — CHANGES_REQUESTED** [review]

File: src/components/DashboardWidget/DashboardWidget.vue (hunk 1 above)
Comment (raimund-schluessler) [reply src/components/DashboardWidget/DashboardWidget.vue:319]: Yes, of course. Good, that we need two reviewers :see_no_evil:

**Review by korelstar — APPROVED** [review]

**Review by skjnldsv — APPROVED** [review]

**Pull request merged**